---
title: Developmental plasticity to high light intensity amplifies the advantage of amphistomous leaves
format:
  pdf:
    keep-tex: true  
    toc: false
author:
  - name: "Christopher D. Muir$^{1,2*}$"
    orcid: 0000-0003-2555-3878
  - name: "Wei Shen Lim$^{1}$"
  - name: "Dachuan Wang$^{1,2}$"
affiliations: false
number-sections: false
header-includes: |
  \usepackage{hyperref}
  \addtokomafont{disposition}{\rmfamily}
  \usepackage{siunitx}
  \newcommand{\aahigh}{\mathrm{AA}_{2000}}
  \newcommand{\aalow}{\mathrm{AA}_{150}}
  \newcommand{\aashade}{\mathrm{AA}_{\text{shade}}}
  \newcommand{\aasun}{\mathrm{AA}_{\text{sun}}}
  \newcommand{\aax}{$\mathrm{AA}$}
  \newcommand{\aay}{\mathrm{AA}}
  \newcommand{\Aamphi}{$A_{\mathrm{amphi}}$}
  \newcommand{\Ahypo}{$A_{\mathrm{hypo}}$}
  \newcommand{\agcurve}{$A \textendash g_\text{sw}$}
  \newcommand{\aqcurve}{$A \textendash Q$}
  \newcommand{\ca}{$C_\mathrm{a}$}
  \newcommand{\caequals}[1]{$C_\mathrm{a} = \qty{#1}{\micro\mol\raiseto{-1}\mol}$}
  \newcommand{\cabetween}[2]{#1 < $C_\mathrm{a} < \qty{#2}{\micro\mol\raiseto{-1}\mol}$}
  \newcommand{\gmaxratio}{$g_\text{max,ratio}$}
  \newcommand{\gmaxshade}{$g_\text{max,shade}$}
  \newcommand{\gmaxsun}{$g_\text{max,sun}$}
  \newcommand{\gsw}{$g_\text{sw}$}
  \newcommand{\loggsw}{$\log(g_\text{sw})$}
  \newcommand{\logA}{$\log(A)$}
  \newcommand{\ppfd}{$\mathrm{PPFD}$}
  \newcommand{\ppfdequals}[1]{$\mathrm{PPFD} = \qty{#1}{\micro\mol\raiseto{-2}\meter\raiseto{-1}\second}$}
  \newcommand{\ppfdqty}[1]{$\qty{#1}{\micro\mol\raiseto{-2}\meter\raiseto{-1}\second}$}
  \newcommand{\rh}{$\mathrm{RH}$}
  \newcommand{\rhequals}[1]{$\mathrm{RH} = #1\%$}
  \newcommand{\tleafequals}[1]{$T_\mathrm{leaf} = \qty{#1}{\degreeCelsius}$}
mainfont: Times New Roman
bibliography: solanum-aa.bib
csl: https://www.zotero.org/styles/science
---

```{=latex}
\begin{center}
$^1$School of Life Sciences, University of Hawaiʻi, Mānoa, Hawaiʻi, USA\\
$^2$Department of Botany, University of Wisconsin, Madison, Wisconsin, USA\\
\medskip
*Corresponding author: cdmuir@wisc.edu\\
\medskip
ORCID: Christopher D. Muir — \href{https://orcid.org/0000-0003-2555-3878}{0000-0003-2555-3878}\\
\end{center}
```

```{r load-objects, echo=FALSE, eval=TRUE, message=FALSE, warning=FALSE}
library(brms)
library(cowplot)
library(dplyr)
library(ggplot2)
library(kableExtra)
library(knitr)
library(lubridate)
library(magrittr)
library(readr)
library(stringr)
library(tibble)
library(tidyr)

theme_set(theme_cowplot())

source("../r/functions.R")

aa_args = read_rds("../objects/aa_args1.rds")
aa_stats = read_rds("../objects/aa_stats.rds")
# sort list by name
aa_stats = aa_stats[order(names(aa_stats))]

plant_info = read_rds("../data/plant-info.rds")

accession_info = read_rds("../data/accession-info.rds") |>
  dplyr::select(species, accession)

rh_curves = read_rds("../data/trimmed_rh_curves.rds")

stomata = read_rds("../data/stomata.rds")

fit_aa1 = read_rds("../objects/fit_aa1.rds")
fit_aa2 = read_rds("../objects/fit_aa2.rds")
dim_fit_aa = dim(fit_aa1$fit)

df_aa_pred1 = read_rds("../objects/df_aa_pred1.rds")
# df_aa_pred2 = read_rds("../objects/df_aa_pred2.rds")
df_aa_pred3 = read_rds("../objects/df_aa_pred3.rds")

aa_loo_table1 = read_rds("../objects/aa_loo_table1.rds")
aa_loo_table2 = read_rds("../objects/aa_loo_table2.rds")


n_plant = length(unique(rh_curves$acc_id))
n_acc = length(unique(rh_curves$acc))
n_sp = accession_info |>
  filter(accession %in% unique(rh_curves$acc)) |>
  pull(species) |>
  unique() |>
  length()

# average number of individual replicates for accession per light treatment
mean_n_rep = rh_curves |>
  summarize(
    n = length(unique(acc_id)),
    .by = c("acc", "light_treatment")
  ) |>
  pull(n) |>
  mean() |>
  prettyNum(digits = 3)

licor_date_range = plant_info |>
  filter(acc_id %in% unique(rh_curves$acc_id)) |>
  dplyr::select(`1s_rh_response_date`, `2s_rh_response_date`) |>
  pivot_longer(cols = everything()) |>
  pull(value) |>
  ymd() |>
  range(na.rm = TRUE) 

n_weeks = prettyNum(as.numeric(difftime(licor_date_range[2], licor_date_range[1], units = "days")) / 7, digits = 3)
 # number of weeks for LICOR measurements
start_licor_date = format(licor_date_range[1], "%B %d, %Y")
end_licor_date = format(licor_date_range[2], "%B %d, %Y")

# Environmental data
df_germ_summary = read_rds("../data/df_germ_summary.rds") |>
  ungroup() |>
  summarize(value = median(value), .by = c("period", "variable", "parameter"))

df_growth_summary = read_rds("../data/df_growth_summary.rds") |>
  ungroup() |>
  filter(!is.na(value)) 

# environmental condition in germination room

germ_temp_day = df_germ_summary |>
  filter(period == "day", variable == "degC", parameter == "mu") |>
  pull(value) |>
  prettyNum(digits = 3)
  
germ_temp_night = df_germ_summary |>
  filter(period == "night", variable == "degC", parameter == "mu") |>
  pull(value) |>
  prettyNum(digits = 3)

germ_rh_day = df_germ_summary |>
  filter(period == "day", variable == "RH", parameter == "mu") |>
  pull(value) |>
  prettyNum(digits = 3)

germ_rh_night = df_germ_summary |>
  filter(period == "night", variable == "RH", parameter == "mu") |>
  pull(value) |>
  prettyNum(digits = 3)

germ_ppfd = df_germ_summary |>
  filter(period == "day", variable == "PAR", parameter == "mu") |>
  pull(value) |>
  prettyNum(digits = 3)

# environmental conditions in growth room

d1 = df_growth_summary |>
  summarize(value = median(value),
            .by = c("period", "rack", "variable", "parameter"))

ppfd_sun = d1 |>
  filter(period == "day", rack == "lower", variable == "PAR", parameter == "mu") |>
  pull(value) |>
  prettyNum(digits = 3)

ppfd_shade = d1 |>
  filter(period == "day", rack == "upper", variable == "PAR", parameter == "mu") |>
  pull(value) |>
  prettyNum(digits = 3)

# degrees warmer that the sun treatment air was
temp_diff = df_growth_summary |>
  filter(period == "day", rack != "hobo", variable == "degC", parameter == "mu") |>
  pivot_wider(names_from = "rack") |>
  mutate(diff = lower - upper) |>
  summarize(value = median(diff, na.rm = TRUE)) |>
  pull(value) |>
  prettyNum(digits = 3)

# difference in RH
rh_diff = df_growth_summary |>
  filter(period == "day", rack != "hobo", variable == "RH", parameter == "mu") |>
  pivot_wider(names_from = "rack") |>
  mutate(diff = upper - lower) |>
  summarize(value = median(diff, na.rm = TRUE)) |>
  pull(value) |>
  prettyNum(digits = 3)

df_leaf_temp = plant_info |>
  filter(!is.na(leaf_temperature_C)) |>
  summarize(
    leaf_temperature_C = mean(leaf_temperature_C),
    n = n(),
    .by = "light_treatment"
  )

# difference in leaf temperature between treatments
leaf_temp_diff = df_leaf_temp$leaf_temperature_C |>
  diff() |>
  prettyNum(digits = 3) 

# number of leaves for which we have measured leaf temp under ambient conditions
n_leaf_temp = sum(df_leaf_temp$n)

# data on leaflet type
df_leaflet_type = plant_info |>
  summarize(n = n(), .by = "leaflet") |>
  pivot_wider(names_from = "leaflet", values_from = "n") |>
  mutate(
    percent_terminal = prettyNum(terminal / (terminal + lateral) * 100, digits = 3),
    percent_lateral = prettyNum(lateral / (terminal + lateral) * 100, digits = 3)
  )

percent_terminal = df_leaflet_type$percent_terminal
percent_lateral = df_leaflet_type$percent_lateral
# percent of time that anatomical traits were measured on different leaflet than LICOR measurements
percent_change_leaflet = full_join(
  plant_info |>
    filter(acc_id %in% unique(rh_curves$acc_id)) |>
    select(acc_id, licor = leaflet),
  stomata |>
    mutate(acc_id = str_c(accession, replicate, sep = "-")) |>
    filter(acc_id %in% unique(rh_curves$acc_id)) |>
    select(acc_id, stomata = leaflet),
  by = join_by(acc_id)
) |>
  summarize(x = mean(licor != stomata, na.rm = TRUE)) |>
  pull(x) |>
  multiply_by(100) |>
  prettyNum(digits = 3)

 # percent of times where AA and AQ curves were done on same leaflet
percent_aa_and_aq = plant_info |>
  filter(acc_id %in% unique(rh_curves$acc_id)) |>
  mutate(same_leaflets = !switch_leaflets) |>
  pull(same_leaflets) |>
  mean() |>
  multiply_by(100) |>
  prettyNum(digits = 3)

# data on chamber light conditions
percent_red = rh_curves |>
  mutate(p_red = Q_red / Q) |>
  summarize(p_red = mean(p_red, na.rm = TRUE),
            .by = c(acc_id, light_intensity)) |>
  summarize(p_red = mean(p_red, na.rm = TRUE), .by = light_intensity) |>
  pivot_wider(names_from = light_intensity, values_from = p_red)

p_red_high = format(signif(100 * percent_red$`2000`, 3), nsmall = 1)
p_blue_high = format(signif(100 * (1 - percent_red$`2000`), 3), nsmall = 1)
p_red_low = format(signif(100 * percent_red$`150`, 3), nsmall = 1)
p_blue_low = format(signif(100 * (1 - percent_red$`150`), 3), nsmall = 1)

# percent measure LMA on alternate leaflet
percent_lma_switch = plant_info |>
  filter(acc_id %in% rh_curves$acc_id) |> 
  summarize(x = mean(leaflet != lma_leaflet, na.rm = TRUE)) |>
  pull(x) |>
  multiply_by(100) |>
  prettyNum(digits = 3)

 # percent leaves missing LMA
percent_lma_missing = plant_info |>
  filter(acc_id %in% rh_curves$acc_id) |> 
  summarize(x = mean(is.na(lma_leaflet))) |>
  pull(x) |>
  multiply_by(100) |>
  prettyNum(digits = 3)

### Cleaning RH curves
ca_min = aa_args$ca_min
ca_max = aa_args$ca_max
n_min_points_per_rh_curve = aa_args$n_min_points_per_rh_curve
point_outlier_threshold = aa_args$point_outlier_threshold
aa_outlier_threshold = aa_args$aa_outlier_threshold
thinning_interval = aa_args$thinning_interval

### Cleaning AQ curves
aq_r2_threshold = aa_args$aq_r2_threshold

### Bayesian data analysis
n_model1 = nrow(aa_loo_table1)
n_model2 = nrow(aa_loo_table2)

### Summarizing AA results
aa_mean_range = df_aa_pred1 |>
  ungroup() |>
  mutate(signif = sign(.lower) == sign(.upper)) |>
  summarize(
    min = min(aa),
    max = max(aa),
    aa = mean(aa),
    n_signif = sum(signif),
    n_acc = n(),
    .by = c(light_treatment, light_intensity)
  ) |>
  mutate(x = sprintf(
    "%.3f (range: %.3f–%.3f; %.0f of %.0f populations significant)",
    aa,
    min,
    max,
    n_signif,
    n_acc
  ))

aa_summary3 = df_aa_pred3 |>
  mutate(signif = (.upper < 0)) |>
  summarize(n_signif = sum(signif), .by = light_treatment) |>
  pull(n_signif, light_treatment)

# Effect of light treatment on LMA
post_lma = posterior_summary(fit_aa2)["b_llma_light_treatmenthigh", ]


```

## Abstract

The presence of stomata on both leaf surfaces (amphistomy) can increase photosynthesis in C$_3$ plants by reducing the path length for CO$_2$ diffusion between substomatal cavities and chloroplasts. Amphistomatous leaves are common among herbaceous plants growing in sunny habitats, including many crop species. This distribution of amphistomatous leaves in nature may result from an increased photosynthetic benefit of amphistomy under high light intensity, either because of acclimatory, plastic, or constitutive variation in CO$_2$ supply or demand. We used a recently developed method to quantify amphistomy advantage, the photosynthetic rate of an amphistomatous leaf relative to an otherwise identical hypostomatous leaf, in `r n_acc` diverse populations representing `r n_sp` species of wild tomatoes (*Solanum* sect. *Lycopersicon* and sect. *Lycopersicoides*). Plant grown under high light intensity benefit more in terms of CO$_2$ assimilated for a given stomatal conductance than plants grown under low light, regardless of measurement light intensity or native light habitat. Developmental plasticity in leaf thickness and/or density, an adaptive response to optimize light interception, likely affects CO$_2$ diffusion within the leaf, altering amphistomy advantage as a byproduct. Developmental plasticity may therefore play an important and previously unnoticed role in explaining the adaptive significance of amphistomy and the distribution of amphistomatous leaves in nature.

## Main text

<!-- Paragraph 1 -->
Stomata are microscopic pores on the surfaces of leaves and other photosynthetic organs formed by a pair of guard cells. They are essential for balancing carbon gained per unit water lost and permitted vascular plants to grow tall on land by enabling access to CO$_2$ for photosynthetic carbon assimilation while preventing hydraulic failure in variable environments [@raven_selection_2002; @mcadam_stomata_2021; @clark_origin_2022]. Optimal stomatal function depends on both dynamic changes in aperture on the scale of minutes to hours, as well as static anatomy determined by developmental plasticity and constitutive genetic differences [@hetherington_role_2003; @de_boer_optimal_2016; @harrison_influence_2020]. Understanding how stomata respond to environmental change over daily, developmental, and evolutionary time is important for understanding adaptation [@woodward_stomatal_1987; @buckley_modelling_2013; @raven_selection_2002; @haworth_co-ordination_2013; @liang_stomatal_2023; @chua_stomatal_2024; @lang_century-long_2024], predicting paleoclimate from fossil cuticles [@franks_new_2014; @mcelwain_paleoecology_2017; @the_cenozoic_co_proxy_integration_project_cencopip_consortium_toward_2023], and improving crops [@hofmann_impact_2025]. Stomatal function contributes to global carbon and water cycles [@berry_stomata:_2010] and therefore predicting future climate [@franks_stomatal_2017].

<!-- Paragraph 2 -->
<!-- Parkhurst gradients 1988 -->
<!-- Syvertsen et al 1995 - may be more here to check about gi in sun and shade-->
<!-- "Farquhar and Raschke (2). However, using a technique similar to ours, Sharkey et al. (11) estimate r, in Xan:hium to be closer to 1.0 m2 s mol-', and state that they believe the estimate of 3.2 m2 s mol-' to be high" -->
Despite extensive theoretical and empirical progress understanding stomata function and anatomy from molecular to ecosystem levels, the adaptive significance of amphistomatous leaves remains an important unsolved problem in leaf structure-function relationships [@grubb_leaf_1977; @parkhurst_adaptive_1978; @mott_adaptive_1982; @gibson_structure-function_1996; @smith_leaf_1997; @oguchi_leaf_2018; @drake_two_2019; @grubb_leaf_2020]. Amphistomatous leaves develop abaxial and adaxial stomata whose aperture can be independently regulated [@pospisilova_environmental_1980; @mott_stomatal_1984; @reich_effects_1985; @mott_asymmetric_1993; @wall_stomata_2022] to control gas exchange through each surface. All else being equal, gas exchange through stomata on both surfaces increases CO$_2$ supply to chloroplasts by providing a second parallel pathway through leaf intercellular airspaces, enhancing photosynthesis [@parkhurst_adaptive_1978; @gutschick_photosynthesis_1984]. The extent to which amphistomy increases CO$_2$ supply depends on resistance to diffusion in intercellular airspaces. This resistance can be low in thin, porous, amphistomatous leaves [@mott_stomatal_1984; @marquez_assessing_2023], but may be more substantial in thick, dense, hypostomatous leaves [@parkhurst_intercellular_1990]. Amphistomatous leaves also lose more water through evaporation because of a second boundary layer conductance [@foster_influence_1986], but the additional carbon gain should be enough to offset this cost in most realistic scenarios [@muir_is_2019]. 

<!-- Paragraph 3 -->
The paradoxical fact is that, despite the photosynthetic benefit, most leaves are not amphistomatous. Many vertically oriented and/or isobilateral leaves are amphistomatous [@wood_physiology_1934; @howell_concerning_1945, @drake_two_2019]. But among dorsiventral leaves, herbaceous plants in open, high light habitats tend to have have amphistomatous leaves [@salisbury_i_1928; @peat_comparative_1994; @gibson_structure-function_1996; @jordan_using_2014; @bucher_stomatal_2017; @muir_light_2018; @triplett_stomatal_2025]. Most other leaves, except those from aquatic habitats, are hypostomatous, producing stomata only on the lower, abaxial surface. Even resupinate leaves develop stomata on the lower, albeit adaxial surface [@lyshede_comparative_2002], suggesting that leaf orientation (lower vs. upper) rather than leaf polarity (abaxial vs. adaxial) is causal. The covariation between stomatal density ratio and light habitat is both qualitative and quantitative. A higher proportion of sun leaves are amphistomatous [@salisbury_i_1928] and the proportion of stomata on the upper, adaxial surface increases with light [@bucher_stomatal_2017; @muir_light_2018]. Resolving why high light intensity favors amphistomatous dorsivental leaves is an important first step toward understanding variation in stomatal density ratio and leaf structure-function relationships more generally.

<!-- Paragraph 4+, hypotheses and predictions -->
The overarching hypothesis is that leaves with greater stomatal density ratio are more common in open, sunny habitats because they increase photosynthesis most in those circumstances. An amphistomatous leaf increases photosynthetic carbon gain compared to an otherwise identical hypostomatous leaf by increasing conductance through the leaf intercellular airspaces and boundary layers; the additional water loss through a second boundary layer is typically small [@foster_influence_1986]. We quantify this benefit as the amphistomy advantage (\aax{}), the log-response ratio of photosynthesis in an amphistomatous leaf compared to an otherwise identical pseudohypostomatous leaf [@parkhurst_adaptive_1978; @triplett_amphistomy_2024]. Why would \aax{} be greater in sun than shade? We consider three nonmutually exclusive hypotheses that we classify as 'acclimatory', 'plastic', and 'constitutive' ([Fig. @fig-concept]).

*Acclimatory hypothesis*: Photosynthetic induction to high light intensity typically involves increases in total leaf stomatal conductance (increased CO$_2$ supply), the concentration of active Rubisco, and electron transport capacity (increased CO$_2$ demand). A one-dimensional circuit model using the Farquhar-von Caemmerer-Berry biochemical model of C$_3$ photosynthesis [@farquhar_biochemical_1980] shows that both increased stomatal conductance and Rubisco activity should increase \aax{}, all else being equal (Supporting Information). If the acclimatory hypothesis is correct, we predict that $\aahigh{} > \aalow{}$ for all populations regardless of native habitat or growth environment. 

*Plasticity hypothesis*: Individuals of the same genotype often develop dramatically different leaves in sun and shade conditions [@poorter_metaanalysis_2019]. Plastic responses are likely adaptations to optimize photosynthesis at different light intensities in variable environments [@givnish_common-garden_2014]. Plastic changes in leaf anatomy and biochemistry could modulate \aax{} as a byproduct. Thicker or less porous leaves, both of which are associated with high leaf mass per area (LMA), will have lower $g_\mathrm{ias}$; leaves with increased total stomatal density and photosynthetic capacity have greater potential CO$_2$ supply and demand. Under the plastic hypothesis, we predict that $\aasun{} > \aashade{}$ for all populations and light intensities. Secondarily, $\aasun{}$ and \gmaxsun{} should also be positively associated with native light habitat if genotypes adapted to sunny, open habitats if they can express a phenotype best adapted to that environment when leaves develop under high light intensity. 

*Constitutive hypothesis*: In environments that are relatively constant or where environmental change cannot be anticipated by a reliable cue, natural selection will favor constitutive expression of optimal phenotypes. We therefore predict genotypes from more sunny, open habitats will have consistently greater \aax under also measurement and growth light intensities. For herbaceous plants, light intensity is largely a function of the tree canopy [@engelbrecht_evaluation_2001]. Herbs growing in the open will regularly experience high light intensity; herbs growing under a forest canopy will often experience low light intensity.

```{r, concept, echo = FALSE}
#| label: fig-concept
#| fig-align: center
#| fig-cap: "\\textbf{Conceptual outline of three nonmutually exclusive hypotheses explaining why amphistomy advantage (\\aax{}) might be greater for leaves in sunny, open habitats.} The acclimatory hypothesis predicts that \\aax{} is greater under high measurement light intensity (\\ppfdqty{2000}) than low measurement light intensity (\\ppfdequals{150}), regardless of growth light intensity or native plant area index (PAI). The plasticity hypothesis predicts that \\aax{} is greater for plants grown in sun than shade, regardless of measurement light intensity or native PAI. The constitutive hypothesis predicts that \\aax{} is greater for plants adapted to sunny habitats (lower native PAI) than shaded habitats, regardless of measurement light intensity or growth light intensity."

knitr::include_graphics("../figures/amphisomy draft 2024-03-28.jpg")
```

The primary directional predictions for each hypothesis are summarized in @tbl-hypotheses; detailed predictions for results that would indicate support for multiple hypotheses are in @tbl-predictions.

```{r, hypotheses, echo = FALSE}
#| label: tbl-hypotheses
#| tbl-cap: "Three nonmutually exclusive hypotheses and directional predictions explaining why amphistomy advantage (\\aax{}) might be greater for leaves in sunny, open habitats. For each hypothesis, we make predictions for how measurement light intensity (\\ppfdequals{150} vs. \\ppfdqty{2000}), growth light intensity (sun vs. shade), and native plant area index (PAI) would affect \\aax{}. \\ppfd{}: photosynthetic photon flux density."

kableExtra::kable(
  data.frame(
    hypothesis = c("acclimatory", "plastic", "constitutive"),
    measurement = c(
      "$\\mathrm{AA}_{2000} > \\mathrm{AA}_{150}$",
      "$\\mathrm{AA}_{2000} = \\mathrm{AA}_{150}$",
      "$\\mathrm{AA}_{2000} = \\mathrm{AA}_{150}$"
    ),
    growth = c(
      "$\\mathrm{AA}_{\\text{sun}} = \\mathrm{AA}_{\\text{shade}}$",
      "$\\mathrm{AA}_{\\text{sun}} > \\mathrm{AA}_{\\text{shade}}$",
      "$\\mathrm{AA}_{\\text{sun}} = \\mathrm{AA}_{\\text{shade}}$"
    ),
    native = c(
      "$\\operatorname{cor}(\\mathrm{PAI}, \\mathrm{AA}) = 0$",
      "$\\operatorname{cor}(\\mathrm{PAI}, \\mathrm{AA}) = 0$",
      "$\\operatorname{cor}(\\mathrm{PAI}, \\mathrm{AA}) > 0$"
    )
  ),
  col.names = c(
    "Hypothesis",
    "Measurement light intensity",
    "Growth light intensity",
    "Native PAI"
  ),
  escape = FALSE,
  booktabs = TRUE,
  format = "latex",
  align = "c"
) |>
  kable_styling() |>
  column_spec(1, width = "1in") |>
  column_spec(2, width = "1.5in") |>
  column_spec(3, width = "1.5in") |>
  column_spec(4, width = "1.25in")
```

<!-- Paragraph 5 (summarize methods) -->
We tested these hypotheses by comparing \aax{} among amphistomatous wild tomato species [@peralta_taxonomy_2008] from different native light habitats, grown under simulated sun and shade light treatments, and measured under contrasting light intensity (). We measured AA on `r n_plant` individual plants from `r n_acc` populations (average of `r mean_n_rep` replicates per light treatment) using a recently developed method [@triplett_amphistomy_2024]. With this method, we directly compare the photosynthetic rate of an untreated amphistomatous leaf to that of the same leaf with gas exchange blocked through the adaxial (upper) surface by transparent plastic, which we refer to as 'pseudohypostomy'. To compare amphi- and pseudohypostomatous leaves at identical whole-leaf $g_\text{sc}$, we measure $A$ over a range of $g_\text{sc}$, inducing stomatal opening and closure by modulating humidity (see Materials and Methods for further details). We estimated 'amphistomy advantage' (\aax) *sensu* @parkhurst_adaptive_1978, but with modifications previously described in @triplett_amphistomy_2024 and here (Materials and Methods). The native light intensity was represented by plant area index (PAI $\unit{\meter\squared\per\meter\squared}$), estimated using a global gridded data set derived from the Global Ecosystem Dynamics Investigation [GEDI; @burns_multi-resolution_2024] and georeferenced accession collection information from the Tomato Genetics Resource Center. The growth light intensities were \ppfdequals{`r ppfd_sun`} (sun treatment) and \ppfdqty{`r ppfd_shade`} (shade treatment) while all other environment conditions were nearly identical (see materials and methods in supplementary materials). The high and low measurements intensities were \ppfdequals{2000} (`r p_red_high`:`r p_blue_high` red:blue) and \ppfdequals{150} (`r p_red_low`:`r p_blue_low` red:blue), respectively. 

<!-- Paragraph 6 (summarize main results) -->
Consistent with biophysical theory of CO$_2$ diffusion within leaves, $\aay{} > 0$ for all populations ([Fig. @fig-aa]A). Bayesian phylogenetic mixed effects models that allowed \aax{} to vary between measurement light intensities, growth light intensities, and among populations outperformed simpler models based on their information criteria (@tbl-aa_loo1). Measured under high light intensity, \aax{} was consistently greater for sun plants. The average \aax{} among populations in the shade treatment was `r filter(aa_mean_range, light_treatment == "low", light_intensity == "2000")$x`; however, the same populations grown at high light intensity showed a mean \aax{} of `r filter(aa_mean_range, light_treatment == "high", light_intensity == "2000")$x`. Contrary to the predictions of the assimilatory hypothesis, \aax{} was greater in all populations under low measurement light intensity for both sun and shade grown plants. The effect of low light on \aax{} was more pronounced in the sun-grown plants, where \aax{} was significantly greater under low measurement light intensity in `r aa_summary3["high"]` of `r n_acc` populations compared to `r aa_summary3["low"]` populations for shade-grown plants. The overall average \aax{} of shade and sun grown plants measured under low light intensity was `r filter(aa_mean_range, light_treatment == "low", light_intensity == "150")$x` and `r filter(aa_mean_range, light_treatment == "high", light_intensity == "150")$x`, respectively. There was a slight tendency for populations from more closed habitats (greater PAI), but \aax{} varied widely among populations from open habitats (low PAI) regardless of growth and measurement light intensities ([Fig. @fig-aa]B). 

```{r, aa, echo = FALSE, warning=FALSE, message=FALSE} 
#| label: fig-aa
#| fig-align: center
#| fig-width: 6
#| fig-height: 8
#| fig-cap: "Caption on following page."

plot_grid(
  read_rds("../objects/fig_aa.rds"),
  read_rds("../objects/fig_aa_pai.rds"),
  ncol = 1,
  labels = "AUTO",
  label_size = 20,
  label_fontface = "bold",
  align = "hv"
)

```

```{=latex}
\addtocounter{figure}{-1}
```

```{r, aa_cap, echo = FALSE, warning=FALSE, message=FALSE} 
#| label: fig-aacap
#| fig-align: center
#| fig-width: 0.1
#| fig-height: 0.1
#| fig-cap: "\\textbf{Amphistomy advantage (\\aax{}) in wild tomatoes is amplified under simulated sunny growth conditions.} \\aax{} ($y$-axis) is the log-response ratio of photosynthesis in an amphistomatous leaf compared to an otherwise identical pseudohypostomatous leaf. In both panels, estimates are shown for plants grown under simulated shade or sun and measured under high light intensity (\\ppfdequals{2000}) and low light intensity (\\ppfdequals{150}). The dashed line at zero indicates no difference in photosynthesis between amphi- and pseudohypostomatous leaves. (\\textbf{A}) The points are the posterior median \\aax{} for each population, with error bars showing 95% confidence intervals. Within each facet, the populations are arranged by \\aax{} estimated in that growth and measurement condition. (\\textbf{B}) The same estimates of \\aax{} in each combination of growth and measurement light intensity ploted against native plant area index (PAI; $x$-axis, log-scale). The confidence intervals were omitted for visual clarity."


ggplot()
```



<!-- Paragraph 7 (summarize secondary results) -->
The pattern of \aax{} across wild tomatoes strongly supports the plasticity hypothesis, argues against the acclimatory hypothesis, and provides only weak support for the constitutive hypothesis. Plastic changes in leaf thickness and/or packing density, summarized by the bulk leaf mass per area (LMA), likely mediate the effect of growth light intensity on \aax{}. LMA increased in sun grown plants in all populations by an average of `r signif(100 * (exp(post_lma["Estimate"]) - 1), 3)`% [95% CI: `r signif(100 * (exp(post_lma["Q2.5"]) - 1), 3)` to `r signif(100 * (exp(post_lma["Q97.5"]) - 1), 3)`%], similar to plastic responses in many species [@poorter_metaanalysis_2019]. When LMA is included in the model, the effect of growth light intensity on \aax{} is no longer significant based on model comparison (Table S8) and LMA is weakly positively correlated with \aax{} ([Fig. @fig-lma_aa]). In contrast, LMA did not mediate an indirect effect of native PAI on \aax{} (Table S8). Many anatomical traits underlie LMA [@john_anatomical_2017] and future research will be needed identify which particular traits, such as leaf thickness or mesophyll porosity, are responsible for mediating \aax{}. The fact that the \aax{} of sun plants was greater under low measurement light intensity supports a long-standing hypothesis that resistance to CO$_2$ diffusion is greater in the upper than lower portions of the leaf interior [@jones_effects_1972]. At low light intensity, the bulk properties of the leaf are weighed toward the upper palisade where most light is intercepted. Hence, amphistomy may, expectedly, be particularly beneficial for sun leaves experiencing intermittant shade or cloud cover. Our study is limited in testing this because we could not directly measure the stomatal conductance ratio and intercellular resistance on each surface. Future experiments measuring \aax{} with a dual sided chamber [@wall_stomata_2022; @marquez_assessing_2023] can overcome these limitations.

```{r, lma_aa, echo = FALSE,warning=FALSE, message=FALSE} 
#| label: fig-lma_aa
#| fig-align: center
#| fig-width: 6
#| fig-height: 4
#| fig-cap: "\\textbf{Plastic changes in leaf mass per area (LMA) mediate the effect of sun- to shade-grown plants on amphistomy advantage (\\aax{}).} Each point is the estimated population-level LMA ($x$-axis) and \\aax{} ($y$-axis) for each population grown under high light intensity (\\ppfdequals{2000}) and low light intensity (\\ppfdequals{150}). The points are colored by growth light treatment. The solid line is the posterior median of the linear regression of individual-level LMA on $\\aahigh$  with the shaded region showing the 95% confidence ribbon; the dashed line is the same for $\\aalow$. Confidence intervals for each population estimate and individual-level estimates are omitted for visual clarity."

read_rds("../objects/fig_lma_aa.rds")

```

We conclude that developmental plasticity, as opposed to acclimation or adaptation to open habitats, may explain the explain the long-standing observation that amphistomatous leaves are more common in sunny habitats, at least among herbaceous plants including crop relatives. Our results advance existing explanations by showing high light intensity *per se* does not increase the benefit of amphistomy, but rather that anatomical and biochemical change associated with higher light intensity modulate \aax{}. Developmental plasticity needs to be considered when interpreting phylogenetic comparative analyses of adaptation, because plastic differences may be mistaken for genetic differences.

maybe this wording is better: Amphistomy likely plays a significant, but under appreciated, role in the carbon and water balance of plant groups where this trait is common, such as herbaceous plants in sunny habitats and woody species in arid biomes (cites). If our estimates of AA from the wild tomato clades

note: estimates of AA are less than that described by Parkhurst and Mott (1990) and also less than that of Xiong and Flexas (2021). What does this tell us?

A second major conclusion is that the significant photosynthetic gain of amphistomatous leaves implies that the many hypostomatous leaves that dominate mesic to wet forests globally are giving up 'free' carbon that would require little to no additional water loss. Under realistic assumptions, we estimate that hypostomatous leaves could achieve the same $A$ with X to X% less water loss if they were amphistomatous (Table S9). Ours is the first experimental demonstration that amphistomy is beneficial for sun leaves because of plastic changes, but future research needs to estimates the tradeoffs associated with amphistomatous leaves in order to fully explain their rarity among many woody plants and shade-tolerant herbs.

## Acknowledgements

Sam McKlin and Tom Buckley helped with protocol development. Justin Alter, Max Gatlin, Joana Kim, Jenna Matsuyama, Brandon Najarian, and Kai Yasuda contributed to data collection. Sarah Friedrich helped with conceptual figure.

### Funding:

US National Science Foundation OIA-1929167 to C.D.M.

### Author contributions:

Conceptualization: C.D.M.; Methodology: C.D.M., W.S.L.; Investigation: C.D.M., W.S.L., D.W.; Visualization: C.D.M.; Funding acquisition: C.D.M.; Writing – original draft: C.D.M.; Writing – review & editing: C.D.M., W.S.L., D.W.

## References

::: {#refs}
:::

{{< pagebreak >}}

# Supplementary Materials

```{=latex}
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\theequation}{S\arabic{equation}}
\setcounter{figure}{0}
\setcounter{table}{0}
\setcounter{equation}{0}
```

## Materials and Methods {#sec-methods}

### Populations

We compared \aax{} among `r n_acc` ecologically diverse populations of wild tomato, including representatives of all described species of *Solanum* sect. *Lycopersicon* and sect. *Lycopersicoides* [@peralta_taxonomy_2008] and the cultivated tomato *S. lycopersicum* var. *lycopersicum* (@tbl-populations). Due to constraints on growth space and time, we spread out measurements over `r n_weeks` weeks from `r start_licor_date` to `r end_licor_date`. Replicates within population were evenly spread out over this period to prevent confounding of temporal variation in growth conditions with accession. [anything else to say here? maybe explain population selection and phylogeny?]

```{r, populations, eval=TRUE,echo=FALSE,message=FALSE,warning=FALSE}
#| label: tbl-populations
#| tbl-cap: "Accession information of *Solanum* populations used in this study. The species name, accession number, collection latitude, longitude, elevation, and daily solar radiation estimated using the SPLASH algorithm (see 'Climate data'). TGRC: Tomato Genetics Resource Center; PAI: Plant area index."

# Copied from solanum-aq project

accession_info = read_rds("../data/accession-info.rds") |>
  filter(accession %in% rh_curves$acc) |>
  left_join(read_rds("../data/accession-gedi.rds") |>
              select(accession, pai), by = join_by(accession)) |>
  arrange(species, accession) |>
  select(
    Species = species,
    `TGRC accession` = accession,
    `Latitude` = lat,
    `Longitude` = lon,
    `Elevation (mas)` = elev,
    `$\\mathrm{PAI}~(\\unit{\\meter\\squared\\per\\meter\\squared})$` = pai
  )

accession_info |>
  knitr::kable(
    format = "latex",
    digits = c(rep(0, 2), 3, 3, 0, 2),
    booktabs = TRUE,
    longtable = TRUE,
    escape = FALSE
  ) |>
  kableExtra::kable_styling(latex_options = c("striped", "no_vertical_lines", "repeat_header")) |>
  kableExtra::column_spec(1, width = "3.25cm", italic = TRUE)

```

### Plant growth conditions

In all growth spaces, we recorded $\mathrm{PPFD}$ using full spectrum quantum sensors (SQ-500-SS, Apogee Instruments, Logan, Utah, USA); we recorded temperature, RH, and [CO$_2$] using an EE894 sensor (E+E Elektronik, Engerwitzdorf, Austria) protected by a radiation shield. All environmental measurements were taken every 10 minutes from the middle of plants racks at approximately the same height as the leaves we measured. We measured leaf temperature of focal leaves prior to measurement using an infrared radiometer (SI-111-SS, Apogee Instruments, Logan, Utah, USA).

#### Germination and seedling stage

Seeds provided by the Tomato Genetics Resource Center germinated on moist paper in plastic boxes after soaking for 30-60 minutes in a 50% (volume per volume) solution of household bleach and water, followed by a thorough rinse. We transferred seedlings to cell-pack flats containing Pro-Mix BX potting mix (Premier Tech, Rivière-du-Loup, Quebec, Canada) once cotyledons fully emerged, typically within 1-2 weeks of sowing. We grew seeds and seedlings for both sun and shade treatments under the same environmental conditions (12:12 h, `r germ_temp_day`:$\qty{`r germ_temp_night`}{\degreeCelsius}$, `r germ_rh_day`:`r germ_rh_night` RH day:night cycle). LED light provided $\mathrm{PPFD} = \qty{`r germ_ppfd`}{\micro\mol\raiseto{-2}\meter\raiseto{-1}\second}$ (Fluence RAZRx, Austin, Texas, USA).

#### Light treatments

Seedlings were randomly assigned in alternating order within population to the sun or shade treatment during transplanting. After seedlings established in cell-pack flats for $\approx 2$ weeks, we transplanted them to 3.78 L plastic pots containing 60% Pro-Mix BX potting mix, 20% coral sand (Pro-Pak, Honolulu, Hawaiʻi, USA), and 20% cinders (Niu Nursery, Honolulu, Hawaiʻi, USA). Percentage composition is on a volume basis. The soil mixture contained slow release NPK fertilizer following manufacturer instructions (Osmocote Smart-Release Plant Food Flower & Vegetable, The Scotts Company, Marysville, Ohio, USA). We determined pot field capacity one week after transplanting using a scale (Ohaus V12P15 Valor 1000, Parsippany, New Jersey, USA) and watered to field capacity three times per week to prevent drought stress. 

We assigned sun and shade treatment to lower and upper racks of a $\qty{1.22}{\meter} \times \qty{2.44}{\meter}$ shelving unit in a climate-controlled growth room. We assigned the sun treatment to the lower rack to limit diffuse light from reaching the shade treatment. The average daytime \ppfd{} was \ppfdqty{`r ppfd_sun`} and \ppfdqty{`r ppfd_shade`} for sun and shade treatments, respectively. To isolate the effect of light intensity from quality, we used the same LED model with the the same spectrum (Fluence SPYDR 2i, Austin, Texas, USAS), but dimmed the lights in the shade treatment. To maintain homogeneous environmental conditions other than light, we mixed air within the growth room using an air circulator (Vornado 693DC, Andover, Kansas, USA) and within racks using a miniature oscillating air circulator (Vornado Atom 1, Andover, Kansas, USA). Despite these efforts, the air in the sun treatment was on average $\qty{`r temp_diff`}{\degreeCelsius}$ warmer and the average RH was consequently `r rh_diff`% lower. However, because of evaporative cooling, the leaves in the sun treatment were only $\qty{`r leaf_temp_diff`}{\degreeCelsius}$ on average ($n = `r n_leaf_temp`$ leaves).

### Leaf trait measurements

We selected a fully expanded, unshaded leaf at least six leaves above the cotyledons during early vegetative growth. This typically meant that plants had grown in light treatments for $\approx$ 4 weeks, ensuring they had time to sense and respond developmentally to the light intensity of the treatment rather than the seedling conditions [@schoch_dependence_1980]. Shade plants grew slower than sun plants, hence leaves at the same developmental stage were measured on chronologically older plants in the shade treatment. In some sun plants, we had to use leaves higher on the stem because short internodes made lower leaves inaccessible with the gas exchange equipment. We measured terminal leaflets in `r percent_terminal`\% of cases, but used the lateral leaflet closest to the terminal leaflet when it was damaged or difficult to clamp into the gas exchange chamber. When a leaflet was damaged during gas exchange measurements, we collected anatomical data from the nearest leaflet on the same leaf (`r percent_change_leaflet`)\%. We measured chlorophyll concentration index (CCI) using a chrolophyll concentration meter (MC-100, Apogee Instruments, Logan, Utah, USA) on the lamina of focal leaflets before gas exchange measurements at the same time we measured leaf temperature.

#### Amphistomy advantage

We estimated 'amphistomy advantage' (\aax) *sensu* @parkhurst_adaptive_1978, but with modifications previously described in @triplett_amphistomy_2024. \aax{} is calculated as the log-response ratio of $A$ compared at the same total \gsw:

$$\mathrm{AA} = \mathrm{log}(A_{\mathrm{amphi}} / A_{\mathrm{hypo}})$$

We measured the photosynthetic rate of an untreated amphistomatous leaf (\Aamphi) over a range of \gsw{} values. We refer to this as an \agcurve{} curve. We compared the \agcurve{} curve of the untreated leaf to the photosynthetic rate of pseudohypostomatous leaf (\Ahypo), which is the same leaf but with gas exchange through the upper surface blocked by a neutral density plastic (propafilm).

We measured \agcurve{} curves using a portable infrared gas analyzer (LI-6800PF, LI-COR Biosciences, Lincoln, Nebraska, USA). Light-acclimated plants were placed under LEDs dimmed to match their light treatment during gas exchange measurements. We estimated the photosynthetic rate ($A$) and stomatal conductance to CO$_2$ (\gsw) at ambient CO$_2$ (\caequals{415}) and \tleafequals{25.0}. The irradiance of the light source in the pseudohypo leaf was higher because the propafilm reduces transmission. To compensate for reduced transmission, we increased incident \ppfd for pseudohypo leaves by a factor 1/0.91, the inverse of the measured transmissivity of the propafilm. We also set the stomatal conductance ratio, for purposes of calculating boundary layer conductance, to 0 for pseudohypo leaves following manufacturer directions.

We collected four \agcurve{} curves per leaf, an amphi (untreated) curve and a pseudohypo (treated) curve at high light-intensity (\ppfdequals{2000}; `r p_red_high`:`r p_blue_high` red:blue) and low light-intensity (\ppfdequals{150}; `r p_red_low`:`r p_blue_low` red:blue). We always measured high light-intensity curves first because photosynthetic downregulation is faster than upregulation in these species. To control for order effects, we alternated between starting with amphi or pseudohypo leaf measurements. Unlike @triplett_amphistomy_2024, preliminary experiments with *Solanum* indicated a strong order effect in that $A$ declined in the second curve. Therefore, we made measurements over two days. On the first day, we measured high and low light-intensity curves for either amphi or pseudohypo leaves; on the second day, we measured high and low light-intensity curves on the other leaf type. 

In all cases, we acclimated the focal leaf to high light (\ppfdequals{2000}) and high relative humidity (\rhequals{70}) until $A$ and \gsw{} reach their maximum. After that, we decreased \rh{} to $\approx 10\%$ to induce rapid stomatal closure without biochemical downregulation. Hence, \Aamphi{} and \Ahypo{} were both measured at low chamber humidity after the leaf had acclimated to high humidity. All other environmental conditions in the leaf chamber remained the same. We logged data until \gsw{} reached its nadir. We then acclimated the leaf to low light (\ppfdequals{150}) and \rhequals{70} before inducing stomatal closure with low \rh and logging data as described above.

#### Light-response (\aqcurve) curves

In `r percent_aa_and_aq`\% of plants, we measured light-response (\aqcurve) curves on the same leaflets as \agcurve{} curves. However, when a leaflet was damaged during \agcurve{} curves, we used the next closest leaflet for \aqcurve{} curves. Leaves acclimated to high light-intensity (\ppfdequals{2000}), ambient CO$_2$ (\caequals{415}), \rhequals{50}, and \tleafequals{25}. After $A$ and \gsw{} stabilized, we measured $A$ at 20 light-intensity levels between $0$ and $\qty{2000}{\micro\mol\raiseto{-2}\meter\raiseto{-1}\second}$ in descending order.

#### Stomatal anatomy

<!-- are there any cases where we did not get stomatal anatomy data? -->

We estimated the stomatal density and size on ab- and adaxial leaf surfaces from all leaves, using guard cell length as a proxy for stomatal size since it proportional to maximum conductance [@sack_developmental_2016]. We made surface impressions of leaf lamina from the same area used for gas exchange measurements using a-silicone impression material (Zhermack elite HD+, light body, fast set, Rovigo, Italy). We applied clear nail polish to make positive replicas of the impression. After nail polish dried, we mounted replicas on a microscope slide using transparent tape [@mott_amphistomy_1991]. We digitized a portion of each leaf surface replica using a brightfield microscope (Leica DM2000, Wetzlar, Germany). We counted and measured guard cell length on all stomata using the FIJI implementation of ImageJ2 version 2.3.0 [@schindelin_fiji_2012], then divided the count by the visible leaf area ($\qty{0.890}{\raiseto{2}\milli\meter}$) to estimate stomatal density.

<!-- 
#### Internal leaf anatomy

Add information on internal leaf anatomy, if using 
-->

#### Leaf mass per area

Leaf mass per area (LMA) is the dry mass divided by the leaflet area. We scanned fresh leaflets on a flat bed scanner (Epson V600, Los Alamitos, California. USA) and measured leaflet area from digital images using the FIJI implementation of ImageJ2 version 2.3.0 [@schindelin_fiji_2012]. We dried leaves for 72 hours at $\qty{74}{\degreeCelsius}$ in a food dehydrator (Cosori CP267-FD, Vesync Co., Anaheim, California, USA) and weighed using a benchtop analytical balance (Ohaus PR64 Analytical Balance, Parsippany, New Jersey, USA). In $`r percent_lma_switch`\%$ we measured LMA on the adjacent leaflet because the focal leaflet was damaged or wilted while making surface impressions and we could not reliably estimate area. LMA data are missing from $`r percent_lma_missing`\%$ of individuals because the area or mass was not recorded at all or recorded incorrectly.

### Cleaning \agcurve{} curves

The raw data set consisted of `r prettyNum(aa_stats$n_rh_curve0, big.mark = ",")` \agcurve{} curves with an average of `r formatC(aa_stats$n_point_per_rh_curve0, digits = 3)` points per curve. Manual curation of a data set this size in a principled, consistent manner is not feasible. Therefore, we automated data cleaning using custom *R* scripts. Cleaning is divided into six sequential steps (\autoref{tbl-cleaning1}).

```{r, cleaning1, echo=FALSE,message=FALSE,warning=FALSE}
#| label: tbl-cleaning1
#| tbl-cap: "Six sequential steps for cleaning \\agcurve{} curves. The rationale and procedure for each step are described in the text. The rightmost columns summarize the number of curves and mean number of points per curve remaining after each step. For reference, there are four possible \\agcurve{} curves per replicate: all combinations of leaf type (amphi or pseudohypo) and light intensity (high or low)."

step_description_col = c(
  "1. remove unreliable and unusable data points",
  "2. remove hysteretic portion of \\agcurve{} curves at low \\gsw{}",
  "3. remove outliers within each \\agcurve{} curve",
  "4. remove replicates with no overlap between amphi and pseudohypo \\agcurve{} curves",
  "5. thin redundant data points within each \\agcurve{} curve",
  "6. trim extreme \\aax{} values"
)

n_curve_col = unlist(aa_stats[str_detect(names(aa_stats), "^n_rh_curve[1-9]$")]) |>
  prettyNum(big.mark = ",")

n_point_col = unlist(aa_stats[str_detect(names(aa_stats), "^n_point_per_rh_curve[1-9]$")])

tibble(
  `Step: description` = step_description_col,
  `Number of curves` = n_curve_col,
  `Number of points per curve` = n_point_col
) |>
  knitr::kable(format = "latex",
               align = c("l", "c", "c"),
               digits = c(0, 0, 1),
               booktabs = TRUE, escape = FALSE) |>
  kableExtra::kable_styling(latex_options = c("striped", "no_vertical_lines")) |>
  kableExtra::column_spec(1, width = "9cm") |>
  kableExtra::column_spec(2, width = "2.5cm") |>
  kableExtra::column_spec(3, width = "2.5cm")

```


#### Remove unreliable and unusable data points

*Rationale*: Unreliable data points consisted of those where chamber [CO$_2$] was unstable and therefore measurements are not biologically meaningful. Unusable data points were those where $A < 0$ because the logarithm of a negative number is undefined.

*Procedure*: We retained data points where \cabetween{`r ca_min`}{`r ca_max`} and $A> 0$.

#### Remove hysteretic portion of \agcurve{} curves at low \gsw{}

*Rationale*: In most \agcurve{} curves, we observed a hysteretic response at low \gsw. After \gsw{} and $A$ declined simultaneously, $A$ increased slightly as \gsw{} continued to decline or stabilize, indicating some leaf acclimation to low \rh. We removed this portion of the curve to focus curve-fitting on the primary domain where $A$ increases monotonically with \gsw{}.

*Procedure*: For each curve, we removed data points after \gsw{} had reached its minimum unless there were fewer than `r n_min_points_per_rh_curve` data points remaining.

#### Remove outliers within each \agcurve{} curve

*Rationale*: Individual outliers within \agcurve{} curves, usually caused by transitory changes in chamber conditions, exert undue leverage on parameter estimates and cause bias and/or low precision in parameter estimates.

*Procedure*: We fit provisional quadratic regressions for each curve using ordinary least squares with the `lm()` function in *R*. We sequentially removed data points with an absolute external studentized residual $> `r point_outlier_threshold`$ until none remained.

#### Thin redundant data points within each \agcurve{} curve

*Rationale*: Data points closely spaced along the \agcurve{} curve provide redundant information and may be highly correlated (i.e. pseudoreplication). This occurred because data was logged at a constant temporal interval, but the rate at which \gsw{} declined was not constant. Thinning reduces parameter estimation bias toward densely sampled regions of the curve which may not be the most biologically informative.

*Procedure*: We retained the maxima and minima \gsw{} for each curve and thinned all but one point per thinning interval of $`r thinning_interval`~\log(\si{\mol\raiseto{-2}\meter\raiseto{-1}\second})$, retaining the point nearest the midpoint of the interval.

#### Remove replicates with no overlap between amphi and pseudohypo \agcurve{} curves

*Rationale*: We could not estimate \aax{} for replicates where amphi and pseudohypo \agcurve{} curves did not overlap.

*Procedure*: We removed replicates where the range of \gsw{} values for amphi and pseudohypo \agcurve{} curves did not overlap.

#### Trim extreme \aax{} values

*Rationale*: Extreme \aax{} values were likely due to measurement error or leaf damage. Since amphi and pseudohypo \agcurve{} curves are measured on consecutive days, a poor calibration or a damaged leaf could cause a large difference in $A$ between days, which would appear as an extreme \aax{} value.

*Procedure*: We provionsally estimated \aax{} for each replicate by integrating over the range of \gsw{} values where amphi and pseudohypo \agcurve{} curves overlap. In this procedure, curve parameters were provisionally estimated using ordinary least squares with the `lm()` function in *R*. We then used point estimates of \aax{} for each replicate as the response variable in a linear model with light treatment, light intensity, population, and all interactions as explanatory variables. This model was also fit using ordinary least squares with the `lm()` function in *R*. We classified  extreme \aax{} values as those with an absolute internal studentized residual $> `r aa_outlier_threshold`$. Because these values likely indicate significant measurement error or leaf damage, we removed \agcurve{} curves at both light intensities if either was classified as extreme.

### Bayesian data analysis with *Stan* {#sec-analysis}

Our analysis pipeline consisted of four main steps:

1. Estimate \agcurve{} curve parameters for each leaf
2. Estimate \aax{} for both measurement light intensities using \agcurve{} curve parameters
3. Estimate the effects of light intensity, light treatment, and population on \aax{} (assimilatory and plasticity hypotheses)
4. Estimate the effects of native light habitat on population-level \aax{} (constitutive hypothesis)

All models were using a Bayesian model with HMC sampling in the probabilistic programming language *Stan* [@stan_development_team_stan_2025] using the *R* package **brms** version `r fit_aa1$version$brms` [@burkner_brms_2017]. We used *CmdStan* version `r fit_aa1$version$cmdstan` and **cmdstanr** version `r fit_aa1$version$cmdstanr` [@gabry_cmdstanr_2025] to interface with *R* version `r getRversion()` [@r_core_team_r:_2025]. We sampled the posterior distribution from a single chain with a minimum of `r aa_args$n_iter_init / 2` iterations after `r aa_args$n_iter_init / 2` warmup iterations. If necessary, we refit models with more iterations to decrease the Gelman-Rubin convergence statistic ($\hat{R}$) [@gelman_inference_1992] to less than $`aa_stats[["max_rhat"]]`$ and the bulk effective sample size (ESS) to greater than $`aa_stats[["min_ess"]]`$ for all model parameters. We also increased the the `adapt_delta` parameter until there were fewer than $`aa_stats[["n_divergent"]]`$ divergent transitions.

#### \agcurve{} curve parameters

We modeled \logA as a quadratic function of \loggsw for each leaf, measured at low and high light intensity, in both amphistomatous (untreated) and pseudohypostomatous (propafilm treatment) states. 

#### \aax{} for each light intensity within leaf using \agcurve{} curve parameters
##### MARKER - add figure of this
We estimated \aax{} for each light intensity by integrating the difference in \logA{} between the amphi and pseudohypo \agcurve{} curves over the range of \gsw{} values where the curves overlap (from $\min(\log(g_\text{sw}))$ to $\max(\log(g_\text{sw}))$).

$$\widehat{\mathrm{AA}} = \int_{\text{min(log(}g_\text{sw}))}^{\text{max(log(}g_\text{sw}))} \text{log}\bigg(\frac{\hat{A}_\text{amphi}(x; \theta_{\text{amphi}})}{\hat{A}_\text{hypo}(x; \theta_{\text{hypo}})}\bigg) dx$$
where $\theta_\text{amphi}$ and $\theta_\text{hypo}$ are the quadratic parameters of the amphi and pseudohypo \agcurve{} curves, respectively. We calculated $\widehat{\mathrm{AA}}$ from each draw of the posterior for the paired amphi and pseudohypo \agcurve{} curves to obtain the posterior distribution of \aax{} for each leaf at each light intensity. We then summarized the posterior distribution of \aax{} for each leaf at each light intensity by the posterior median, which became our estimate of \aax{} for subsequent analyses.

### Bayesian phylogenetic mixed effects models 

We fit Bayesian mixed effects models with phylogenetically structured random effects to test predictions of competing hypotheses about why amphistomy advantage (\aax) might be greater for leaves in sunny, open habitats. We fit all models combinations of measurement light intensity, growth light intensity, and their interaction as both fixed effects and random effects among populations. In other words, we test whether there were main effects of light intensity treatments and/or whether populations varied in their response to light intensity treatments. In all models, we included population as a phylogenetically structured random effect. We crossed those fixed and random effect structures with fixed effects of measurement light intensity and/or growth light intensity on the distributional parameter, $\sigma$, the residual variance in \aax{}. In total, we fit `r n_model1` models, each with a different combination of fixed, random, and distributional effects. We used the RAxML whole-transcriptome concatenated phylogeny based on 2,745 100-kb genomic windows from @pease_phylogenomics_2016. Two of our populations were not in this tree. We used accession LA1044 in place of LA3909, two populations of *S. galapagnse*; LA0750 was added as sister to LA0716, two closely replated populations of *S. pennellii*. The node was placed half-way between the next deepest node. 

NEED TO EDIT THIS: The next sections describe how we fit models in *Stan*, all model parameters and priors, and specific predictions about parameter values for each hypothesis.
Each curve was fit For initial fits, we sampled the posterior distribution from a single chain with `r aa_args$n_iter_init / 2` iterations after `r aa_args$n_iter_init / 2` warmup iterations. After model selection (see below), we refit the selected model with more iterations if necessary to decrease the Gelman-Rubin convergence statistic ($\hat{R}$) [@gelman_inference_1992] to less than $`aa_stats[["max_rhat"]]`$ and the bulk effective sample size (ESS) to greater than $`aa_stats[["min_ess"]]`$ for all model parameters.


#### Fitting models in *Stan* using **brms** interface {#sec-fitting}

We fit Bayesian models with HMC sampling in the probabilistic programming language *Stan* [@stan_development_team_stan_2025] using the *R* package **brms** version `r fit_aa1$version$brms` [@burkner_brms_2017]. We used *CmdStan* version `r fit_aa1$version$cmdstan` and **cmdstanr** version `r fit_aa1$version$cmdstanr` [@gabry_cmdstanr_2025] to interface with *R* version `r getRversion()` [@r_core_team_r:_2025]. For initial fits, we sampled the posterior distribution from a single chain with `r aa_args$n_iter_init / 2` iterations after `r aa_args$n_iter_init / 2` warmup iterations. After model selection (see below), we refit the selected model with more iterations if necessary to decrease the Gelman-Rubin convergence statistic ($\hat{R}$) [@gelman_inference_1992] to less than $`aa_stats[["max_rhat"]]`$ and the bulk effective sample size (ESS) to greater than $`aa_stats[["min_ess"]]`$ for all model parameters.







##### Effects of light intensity, light treatment, and population on \aax

We tested for effects of light intensity, light treatment, population, and their interactions on \aax. All models included effects of light intensity and light treatment, as well as random effects of population and replicate within population. More complex models included interactions between light intensity and light treatment, as well as random effects of population on the effects of light intensity and light treatment. We used a weakly informative normal prior with mean $0$ and standard deviation $10$ for fixed effects of light intensity and treatment. We used a weakly informative normal prior with mean $-3$ and standard deviation $5$ for the the random effect standard deviation of replicate within population. We accounted for the phylogenetic structure among the random effects of population using an Ornstein-Uhlenbeck (OU) process [@hansen_stabilizing_1997]. The expected covariance between populations $i$ and $j$ ($\text{Cov}(i, j)$ ) is:

$$\text{Cov}(i, j) = \frac{\sigma^2}{2 \alpha} \exp(-\alpha D_{ij})$$
where $\sigma^2$ is the variance of the random effect, $\alpha$ is the rate of decay of the covariance with phylogenetic distance, $D_{ij}$, the phylogenetic distance between populations $i$ and $j$. We estimated $\sigma^2 / (2 \alpha)$ and $\alpha$ as a separate parameters and reparameterized them as $\sigma^2$ and $\alpha$. We used a weakly informative normal prior with mean $0$ and standard deviation $10$ on OU parameters. 

We modeled the residual standard deviation of \aax, (i.e. phylogenetically unstructured variation unaccounted for by explanatory variables) on a log-link scale with effects of light intensity and light treatment. We used a weakly informative normal prior with mean $-3$ and standard deviation $5$ on the residual standard deviation intercept and a weakly informative normal prior with mean $0$ and standard deviation $1$ on the effects of light intensity and light treatment on the residual standard deviation.

##### Effects of native light habitat on population-level \aax

We tested a linear effect of native light habitat on population-level \aax. In Models 1 and 2, we used the random intercept of \aax{} as a response variable; in model 3 we used the random intercept plus the random effect of population at high light intensity; in model 4 we used the random intercept plus the random effect of population in the sun treatment; in model 5 we used the random intercept plus the random effects of population at high light intensity and sun treatment. We used a weakly informative normal prior with mean $0$ and standard deviation $1$ for the slope and intercept. We accounted for the phylogenetic structure among the model residuals using an (OU) process. We used a weakly informative normal prior with mean $0$ and standard deviation $10$ on OU parameters. 

#### Predictions {#sec-predictions}

The assimilatory, plastic, and constitutive hypotheses make different predictions about the relationship between \aax{} and light intensity, light treatment, and native \ppfd{} among populations. Since these hypotheses are not mutually exclusive, we describe how we assessed support for one, two, or all three hypotheses simultaneously in \autoref{tbl-predictions}. In general, the assimilatory hypothesis was supported if \aax{} was greater at high light intensity than low light intensity. The plastic hypothesis was supported if \aax{} was greater in sun leaves than shade leaves. The constitutive hypothesis was supported if population-level \aax{} increased with native \ppfd. In interactive models, we only consider positively reinforcing interactions between high light intensity, sun leaves, and native \ppfd{} because these are the only interactions which could explain why amphistomatous leaves are advantageous in high light habitats.

We evaluated predictions using a combination of parameter estimation and model selection. In all cases, we estimated parameters and confidence intervals as described in @sec-fitting. If the 95% confidence intervals for a parameter did not overlap zero, we considered the parameter to be significantly different from zero. We used the leave-one-out cross-validation information criterion (LOOIC) to compare the fit of models (\autoref{tbl-models}) using the *R* package **loo** version `r packageVersion("loo")` [@vehtari_practical_2017] to calculate LOOIC values. Note that LOOIC was calculated only from pointwise likelihood values of the submodel estimating effects of light intensity, light treatments, and population on \aax{} (see @sec-parameters). We considered models with two standard errors of the mean lower LOOIC value to be a better fit to the data; we considered models with LOOIC values within two standard errors of the mean to be have similar support.

```{r, predictions, eval=TRUE, echo=FALSE,message=FALSE,warning=FALSE}
#| label: tbl-predictions
#| tbl-cap: "Predictions of competing hypotheses about the relationship between \\aax{} and light intensity, light treatment, and native \\ppfd{} among populations. The middle column lists specific directional predictions about parameter values and model fit according to leave-one-out cross-validation information criterion (LOOIC), where $\\text{LOOIC}_i$ is the LOOIC value for model $i$. The rightmost column describes the predictions in words and explains how population-level \\aax{} is calculated in the relevant model."

tribble(
  ~"\\textbf{Hypothesis}", ~"\\textbf{Prediction(s)}", ~"\\textbf{Description}",
  "Assimilatory", "$\\beta_{\\mathrm{AA},2000} > 0$", "Average \\aax{} at high light intensity is greater than that at low light intensity",
  "Plastic", "$\\beta_{\\mathrm{AA},\\text{sun}} > 0$", "Average \\aax{} in sun leaves is greater than that in shade leaves",
  "Constitutive", "$\\beta_{\\mathrm{PPFD,AA}} > 0$", "Population-level \\aax{} ($\\mathrm{AA}_\\text{acc}$) increases with native \\ppfd",
  "Constitutive", "", "$\\mathrm{AA}_\\text{acc} = \\beta_{\\mathrm{AA}, 0} + \\vec{\\beta}_{\\mathrm{AA}, \\text{acc}}$",
  "Assimilatory $\\times$ Plastic", "$\\beta_{\\mathrm{AA},2000,\\text{sun}} > 0$", "Average \\aax{} is highest at high light intensity in sun leaves",
  "Assimilatory $\\times$ Plastic", "$\\text{LOOIC}_\\text{1} > \\text{LOOIC}_\\text{2}$", "",
  "Assimilatory $\\times$ Constitutive", "$\\beta_{\\mathrm{AA},2000} > 0$", "Average \\aax{} at high light intensity is greater than that at low light intensity",
  "Assimilatory $\\times$ Constitutive", "$\\beta_{\\mathrm{PPFD,AA}} > 0$", "Population-level \\aax{} at high light intensity ($\\mathrm{AA}_{\\text{acc},2000}$) increases with native \\ppfd",
  "Assimilatory $\\times$ Constitutive", "$\\text{LOOIC}_\\text{1} > \\text{LOOIC}_\\text{3}$", "$\\mathrm{AA}_{\\text{acc},2000} = \\beta_{\\mathrm{AA}, 0} + \\vec{\\beta}_{\\mathrm{AA}, \\text{acc}} + \\vec{\\beta}_{\\mathrm{AA}, 2000, \\text{acc}}$",
  "Plastic $\\times$ Constitutive", "$\\beta_{\\mathrm{AA},\\text{sun}} > 0$", "Average \\aax{} in sun leaves is greater than that in shade leaves",
  "Plastic $\\times$ Constitutive", "$\\beta_{\\mathrm{PPFD,AA}} > 0$", "Population-level \\aax{} in sun leaves ($\\mathrm{AA}_{\\text{acc},\\text{sun}}$) increases with native \\ppfd",
  "Plastic $\\times$ Constitutive", "$\\text{LOOIC}_\\text{1} > \\text{LOOIC}_\\text{4}$", "$\\mathrm{AA}_{\\text{acc},\\text{sun}} = \\beta_{\\mathrm{AA}, 0} + \\vec{\\beta}_{\\mathrm{AA}, \\text{acc}} + \\vec{\\beta}_{\\mathrm{AA}, \\text{sun}, \\text{acc}}$",
  "Assimilatory $\\times$ Plastic $\\times$ Constitutive", "$\\beta_{\\mathrm{AA},2000,\\text{sun}} > 0$", "Average \\aax{} is highest at high light intensity in sun leaves",
  "Assimilatory $\\times$ Plastic $\\times$ Constitutive", "$\\beta_{\\mathrm{PPFD,AA}} > 0$", "Population-level \\aax{} at high light intensity in sun leaves ($\\mathrm{AA}_{\\text{acc},2000,\\text{sun}}$) increases with native \\ppfd",
  "Assimilatory $\\times$ Plastic $\\times$ Constitutive", "$\\text{LOOIC}_\\text{3} > \\text{LOOIC}_\\text{5}$", "$\\mathrm{AA}_{\\text{acc},2000,\\text{sun}} = \\beta_{\\mathrm{AA}, 0} + \\vec{\\beta}_{\\mathrm{AA}, \\text{acc}} + \\vec{\\beta}_{\\mathrm{AA}, 2000, \\text{acc}} + \\vec{\\beta}_{\\mathrm{AA}, \\text{sun}, \\text{acc}}$"

) |>
  knitr::kable(
    format = "latex",
    align = c("l", "l", "l"),
    booktabs = TRUE,
    escape = FALSE,
    longtable = TRUE
  ) |>
  kableExtra::add_indent(positions = seq_len(15), level_of_indent = -1, target_cols = 3) |>
  kableExtra::collapse_rows(1, latex_hline = "major", valign = "top",
                            row_group_label_position = "first") |>
  kableExtra::column_spec(1, width = "1in") |>
  kableExtra::column_spec(2, width = "1.5in") |>
  kableExtra::column_spec(3, width = "3in")

```

## Results

* Variation in stomatal ratio among pops and light treatments
* Table S7: Model comparison LOOIC tables
* Table S8: Model comparison LOOIC tables with LMA
* Table S9: Calculation of alternate scenario where amphi leaves can get same A with less R
* Each A-gs figure with fitted curves, R2, and AA

```{r, aa_loo1, echo=FALSE,message=FALSE,warning=FALSE}
#| label: tbl-aa_loo1
#| tbl-cap: 'Leave-one-out cross-validation information criterion (LOOIC) for models of \\aax{} as a function of light intensity, light treatment, and population. The model with the lowest LOOIC is considered the best fit to the data. The LOOIC column lists the LOOIC value for each model; the $\\Delta$LOOIC column lists the difference between the LOOIC of the corresponding model and the best model (lowest LOOIC). Checkmarks in each row indicate whether the corresponding model included affects of measurement light intensity ("Meas.") or growth light intensity ("Growth") as fixed effects, random effects, or distributional effect on the residual variance. We also test for fixed and random interaction effects ("Inter.") between measurement and growth light intensity. All models included phylogenetically structured random effects of population.'

df1 = aa_loo_table1 |>
  mutate(
    fixed_meas = str_detect(fixed, "light_intensity"),
    fixed_growth = str_detect(fixed, "light_treatment"),
    fixed_intrxn = str_detect(fixed, "\\*"),
    # random_acc = TRUE,
    random_meas = str_detect(random, "light_intensity"),
    random_growth = str_detect(random, "light_treatment"),
    random_intrxn = str_detect(random, "\\*"),
    sigma_meas = str_detect(sigma, "light_intensity"),
    sigma_growth = str_detect(sigma, "light_treatment"),
  ) |>
  dplyr::select(
    LOOIC = looic,
    `$\\Delta$LOOIC` = delta_looic,
    starts_with("fixed_"),
    starts_with("random_"),
    starts_with("sigma_")
  ) |>
  mutate(across(where(is.logical), ~ ifelse(.x, "$\\checkmark$", "")))

c1 = colnames(df1)
groupings = list(
  " " = sum(str_detect(colnames(df1), "LOOIC")),
  "Fixed" = sum(str_detect(colnames(df1), "fixed_")),
  "Random" = sum(str_detect(colnames(df1), "random_")),
  "Distributional" = sum(str_detect(colnames(df1), "sigma_"))
)

c2 = case_when(
  str_detect(c1, "_meas") ~ "Meas.",
  str_detect(c1, "_growth") ~ "Growth",
  str_detect(c1, "_intrxn") ~ "Inter.",
  str_detect(c1, "_acc") ~ "Population",
  .default = c1
)

kbl(
  df1,
  col.names = c2,
  booktabs = TRUE,
  escape = FALSE,
  align = "c",
  longtable = TRUE,
  digits = 2
) |>
  kable_styling(latex_options = c("striped", "repeat_header")) |>
  add_header_above(groupings) |>
  column_spec(1, width = "0.7in") |>
  column_spec(2, width = "0.5in") |>
  column_spec(3:10, width = "0.4in") |>
  column_spec(c(5, 8), border_right = TRUE)

```
